docs/secret-rotation.md

```diff
@@ -0,0 +1,254 @@
+# Secret Rotations &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; **FluxCD**
+
+This brief guide should outline all the steps needed to be able to swap out a set of secrets or credentials with FluxCD GitOps.
+
+## [**Pre-requisites**](#pre-requisites-link)
+#### [Useful Resources](#useful-resources-link)
+#### [Applications & Software](#applications--software-link)
+
+## [**Outline**](#outline-link)
```

Comment:
Let's clean up the formatting of this table of contents please. Use something more like [this](https://github.com/azimuth-cloud/capi-helm-charts/blob/main/charts/openstack-cluster/README.md#contents--) which you can see the raw markdown for [here](https://raw.githubusercontent.com/azimuth-cloud/capi-helm-charts/refs/heads/main/charts/openstack-cluster/README.md).

Reply:
Done